algeria/researcher_profile/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+
+
+class Institution(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+class Department(models.Model):
+    name = models.CharField(max_length=255)
+    
+    def __str__(self):
+        return self.name
+
+class Position(models.Model):
+    name = models.CharField(max_length=255)
+    
+    def __str__(self):
+        return self.name
+
+class Research_Area(models.Model): ####implement fuzzy matching / standardization of names
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+
+class Collaboration_Area(models.Model): ####implement fuzzy matching / standardization of names
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+class Link(models.Model):
+    link = models.URLField(max_length = 255, null=True, blank=True)
+    link_type = models.CharField(max_length=255)
+    link_type = models.CharField(max_length=255)
+
+
+    def __str__(self):
+        return self.name
+
+
+
+class Profile(models.Model):
+    name = models.CharField(max_length=255)
+    image = models.URLField(max_length=255, null=True, blank=True)
+    institution = models.ManyToManyField(Institution)
+    department = models.ManyToManyField(Department)
+    position = models.ManyToManyField(Position)
+    email = models.EmailField
```

Comment:
Does this work without a `()` at the end of it?  I would recommend at least adding a `(min_length=4096)` to it.

Reply:
can we add a max length instead? i don't know how we didn't notice this before, but i think max length is the required argument with EmailField

Reply:
the documention from Django says max_length is what is used

Reply:
oops I meant to put `max_length`!  good catch...yes use max_length!